feature_creation_scripts/arctic_ocean_regions/arctic_ocean_regions.py

```diff
@@ -0,0 +1,213 @@
+                        "tags": tags},
+         "geometry": {
+             "type": "Polygon",
+             "coordinates": [[[lon0, lat0],
+                              [lon1, lat0],
+                              [lon1, lat1],
+                              [lon0, lat1],
+                              [lon0, lat0]]]}})
+    return fc
+
+
+def split_rectangle(lon0, lon1, lat0, lat1, name, author, tags, fcContour):
+    fc = make_rectangle(lon0, lon1, lat0, lat1, name, author, tags)
+
+    fcDeep = fc.difference(fcContour)
+
+    props = fcDeep.features[0]['properties']
+    props['name'] = props['name'] + ' Deep'
+    props['tags'] = props['tags'] + ';Deep'
+    props['zmin'] = -1000.
+    props['zmax'] = -400.
+
+    fcShelf = fc.difference(fcDeep)
+
+    props = fcShelf.features[0]['properties']
+    props['name'] = props['name'] + ' Shelf'
+    props['tags'] = props['tags'] + ';Shelf'
+    props['zmin'] = -1000.
+    props['zmax'] = -200.
+
+    fc.merge(fcDeep)
+    fc.merge(fcShelf)
+
+    return fc
+
+
+def main():
+    author = 'Milena Veneziani'
+    arcticTags = 'Arctic_Ocean;Arctic;Arctic_Basin'
+    proshTags = 'Arctic;Proshutinsky'
+
+    # make a geometric features object that knows about the geometric data
+    # cache up a couple of directories
+    gf = GeometricFeatures('../../geometric_data')
+
+    fc = FeatureCollection()
+
+    # Combine old Barentsz_Sea and White_Sea into new Barents Sea 
+    # feature (Barentsz_Sea and White_Sea features will be removed)
+    fcBS = gf.read('ocean', 'region', ['Barentsz Sea'])
+    fcBS.merge(gf.read('ocean', 'region', ['White Sea']))
+    fcBS = fcBS.combine('Barents Sea')
+    fcBS.to_geojson('BarentsSea.geojson')
+    fc = fcBS
+    props = fc.features[0]['properties']
+    props['tags'] = 'Barents_Sea;Arctic;Arctic_Basin'
+
+    # Include Kara Sea
+    fcKara = gf.read('ocean', 'region', ['Kara Sea'])
+    fc.merge(fcKara)
+
+    # Define triangle between Greenland Sea and Arctic_Ocean
+    # (north of Fram Strait) that is not part of any of the
+    # currently defined Arctic features
+    fc_tmp = gf.read('ocean', 'region', ['Arctic Ocean'])
+    fc_tmp.merge(gf.read('ocean', 'region', ['Lincoln Sea']))
+    fc_tmp.merge(fcBS)
+    fcArctic = make_rectangle(lon0=-36., lon1=20., lat0=86., lat1=79.,
+        name='North of Fram Strait', author=author, tags=arcticTags)
+    fcArctic = fcArctic.difference(fc_tmp)
+
+    # Define full Arctic *but* Barents and Kara Seas
+    fcArctic.merge(gf.read('ocean', 'region', ['Arctic Ocean']))
+    fcArctic.merge(gf.read('ocean', 'region', ['Laptev Sea']))
+    fcArctic.merge(gf.read('ocean', 'region', ['East Siberian Sea']))
+    fcArctic.merge(gf.read('ocean', 'region', ['Chukchi Sea']))
+    fcArctic.merge(gf.read('ocean', 'region', ['Beaufort Sea']))
+    fcArctic.merge(gf.read('ocean', 'region', ['Lincoln Sea']))
+    fcArctic = fcArctic.combine('Arctic Ocean')
+    fcArctic.to_geojson('ArcticOcean.geojson')
```

Comment:
write to geojson file is currently just a debugging line.

Note that I needed to add a triangle north of Fram Strait that is in the old Greenland Sea feature, but it really belongs to the Central Arctic.. (see `fc_tmp` lines above)